app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = current_user.payments
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+  end
+
+  def create
+    @payment = Payment.new(payment_params)
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+
+    if @payment.valid? && @payment.purchase(current_user, @conference, price_in_cents) && @payment.save
```

Comment:
`@payment.save` will return `false` if the payment is not valid. Do we need to have `@payment.valid?` here as well?